### ISSUE TITLE
Remove typed passwords after inactivity

### ISSUE THREAD

**Issue by comradekingu** [andOTP/andOTP#952]
A password typed by the user is left in the input field indefinitely.(?)
This remains the case when multitasking over to a different app.
This makes it possible to forget the app in a state that allows it to be opened if the user thought to do something
else after typing the password.

It doesn't seem like general inactivity in the app locks it, which is maybe a problem the user is more aware of.
Plain removal of typed entries after a while at the password stage seems beneficial anyhow.

The chance of leaving a usable password behind can be mitigated by logging in automatically once the correct password is found,
but that needs a timeout for wrong attempts, beyond an allowed amount of characters,
which needs a minimum amount of characters.

https://github.com/andOTP/andOTP/issues/814 somewhat related.

**Comment by aximut** [comment]
I agree with this, this is counter-intuitive to what an average user would probably assume.
